### PR TITLE
Test created

### PR DESCRIPTION
**Pull request by BELOVEZHIE**

### CODE REVIEW

**Review by MariaMoroz — APPROVED** [review]
lgtm

File: src/test/java/EntityBoardDeleteTest.java [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+        findElement(By.xpath("//a[@href = 'index.php?action=action_list&list_type=table&entity_id=31']")).click();
+        findElement(By.xpath("//button[@class = 'btn btn-round btn-sm btn-primary dropdown-toggle']")).click();
+
+        getWait().until(ExpectedConditions.visibilityOfElementLocated(DELETE_RECORD));
+        findElement(DELETE_RECORD).click();
+
+        findElement(By.xpath("//a[@href = 'index.php?action=recycle_bin']/i")).click();
+    }
+
```
Comment (Jenny-ai) [inline src/test/java/EntityBoardDeleteTest.java:46]: I would divide this method into createRecord() and deleteRecord() or at least renamed to createAndDeleteRecord() - for readability

**Review by Jenny-ai — APPROVED** [review]

File: src/test/java/EntityBoardDeleteTest.java (hunk 1 above)
Comment (BELOVEZHIE) [reply src/test/java/EntityBoardDeleteTest.java:46]: Thanks. But my aim to be merged. I cant do it for a week. Will do refactoring after.)))

**Comment by Jenny-ai** [comment]
please rename PR - it should have User Story or TC name

**Comment by BELOVEZHIE** [comment]
ok

**Comment by BELOVEZHIE** [comment]
> please rename PR - it should have User Story or TC name

DONE

**Pull request merged**